scss/_patterns_navigation.scss

```diff
@@ -14,24 +13,61 @@ $lightness-threshold: 70;
     }
   }
 
+  %navigation-link-responsive-padding-vertical {
+    padding-bottom: $spv-inner--medium;
+    padding-top: $spv-inner--medium;
+  }
+
+  %navigation-link-responsive-padding-left {
+    // follows grid padding to ensure nav items on small breakpoints align with grid padding
+    @media (max-width: $threshold-4-6-col) {
```

Comment:
It's probably question for later, but do we need that many different paddings on different screen sizes?
It creates a lot of CSS in media queries in places where we need to align with the grid.

Especially that medium and large are actually the same:
https://github.com/canonical-web-and-design/vanilla-framework/blob/6e1d68232ce8611eb2aa887972fd8afb4af13ef2/scss/_settings_grid.scss#L24-L25

Reply:
No, I guess small/default would suffice. Will create a new issue and tackle separately.